.github/workflows/nodejs.yml

```diff
@@ -78,20 +79,6 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: npm i -g npm
 
-      - name: Get npm cache directory
-        id: npm-cache-dir
-        run: |
-          echo "::set-output name=dir::$(npm config get cache)"
-
-      - name: Cache Node.js modules
-        id: cache
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.npm-cache-dir.outputs.dir }}
-          key: ${{ runner.os }}-${{ matrix.webpack-version }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.webpack-version }}-node-
-
```

Comment:
We need `webpack-version` here, otherwise cache was wrong

Reply:
if I understand correctly, we probably won't need it because we do an install after deps are restored so it'll do a fresh install for webpack version